### ISSUE TITLE
Sequence_length meaning 

### ISSUE THREAD

**Issue by Harin99** [uzh-rpg/E-RAFT#13]
Hello, i'm implementing the training code of E-Raft.

But there's a **sequence_length variable** in the paper and code, so what exactly does that mean?

If i set a sequence_length = 2, it seemed like 2 sequential data were being loaded on the code. ( like below )
e.g) [ 0, 1 ] -> [ 1, 2 ] -> [ 2, 3 ]

I understood that the **E-Raft network** is a structure that give the good initialization value of flow by continuously handing over the flow estimated by GRU structure to the next sequence for sequential data.

In this process, i would like to ask **how the sequence_length is applied to training process.**

Thank you for your great work!!

**Comment by magehrig** [comment]
Hi @Harin99 

Yes, sequence length greater than 1 implies that we use warm starting. Without checking the code, I think that a sequence length of N should load N+1 samples (e.g. [0,1,2,3,4,5] for N = 5). This is because we need at least two subsequent samples to infer optical flow with E-RAFT.

If you would like to reproduce the results, I suggest first getting the model to train without using the warm-starting loss (e.g. sequence length of 1). That should be much easier to implement, and you will already get very good results.

**Comment by Harin99** [comment]
Thank you for replying me! 

And I have another questions about training using dsec dataset. 

In this page (https://dsec.ifi.uzh.ch/dsec-datasets/download/), I can check that **gt flow is consisted of png.** 
- png : ( 480, 640, 3 )

**If then, how can we generate the loss ( gt - prediction )?** 

Also When i debugged the 'dsec_loader.py', flow_timestamp variable is entered the image timestamps in "image_timestamp.txt". 
**Why should we put the image timestamp value in flow timestamp variable?** 
Is it because the test dataset generate the gt flow as a image and measures the metric?
**If then, When i want to train the model using training dataset(dsec) in above link, should flow timestamp.txt value be entered as the flow timestamp variable?** 

Your reply will help me a lot. Thank you so much:)
